evalai/utils/challenges.py

```diff
@@ -87,7 +87,35 @@ def display_ongoing_challenge_list():
     Displays the list of ongoing challenges from the backend
     """
     url = "{}{}".format(API_HOST_URL, URLS.challenge_list.value)
-    display_challenges(url)
+
+    header = get_request_header()
+    try:
+        response = requests.get(url, headers=header)
+        response.raise_for_status()
+    except requests.exceptions.HTTPError as err:
+        if (response.status_code == 401):
+            validate_token(response.json())
+        echo(err)
+        sys.exit(1)
+    except requests.exceptions.RequestException as err:
+        echo(err)
+        sys.exit(1)
+
+    response = response.json()
+    challenges = response["results"]
+
+    # Filter out past/unapproved/unpublished challenges.
+    challenges = list(filter(lambda challenge:
+                             validate_date_format(challenge['end_date']) > datetime.now() and
+                             challenge["approved_by_admin"] and
+                             challenge["published"],
+                             challenges))
+
+    if len(challenges) is not 0:
```

Comment:
Huh? Why not `!=`? This is kind of an anti-pattern since it is used for object equality rather than numerical equality.

Reply:
@varunagrawal Just a careless mistake, I've rectified.